src/utils/GeometryUtils.ts

```diff
@@ -193,4 +201,8 @@ const gosper = (size) => {
   return points
 }
 
-export { hilbert2D, hilbert3D, gosper }
+export const GeometryUtils = {
```

Comment:
Can we keep the export the same? Why do we need to export `GeometryUtils` as a named object?

Reply:
Looks  like [`drei`](https://github.com/pmndrs/drei/blob/master/.storybook/stories/Line.stories.tsx#L3)  think `three-stdlib` has exported `GeometryUtils` also I think it would be better to align with [`three`](https://github.com/mrdoob/three.js/blob/dev/examples/js/utils/GeometryUtils.js#L184)